Tasks/XcodeV5/task.json

```diff
@@ -94,6 +94,7 @@
                 "8": "Xcode 8",
                 "9": "Xcode 9",
                 "10": "Xcode 10",
+                "11": "Xcode 11",
```

Comment:
I'm assuming since Xcode 11 is already on the machines, all we had to do was add the option in the dropdown and it will automatically use it, correct?

Reply:
Correct.